keras_cv/models/segmentation/segformer/segformer.py

```diff
@@ -165,6 +166,29 @@ def get_config(self):
         )
         return config
 
+    @classmethod
+    def from_preset(
+        cls,
+        preset,
+        num_classes,
+        load_weights=None,
+        **kwargs,
+    ):
+        aliases = {
+            "segformer_b0": "mit_b0",
+            "segformer_b1": "mit_b1",
+            "segformer_b2": "mit_b2",
+            "segformer_b3": "mit_b3",
+            "segformer_b4": "mit_b4",
+            "segformer_b5": "mit_b5",
+        }
+        if preset in aliases:
+            preset = aliases[preset]
+        backbone = MiTBackbone.from_preset(
```

Comment:
Could we just chain to super here? And add the mit_b preset to `backbone_presets`?

To hit this branch
https://github.com/keras-team/keras-cv/blob/f64fba0decee9306874dc5f88aaa6d20538be0d7/keras_cv/models/task.py#L134-L141

That would have the advantage of being totally uniform with how other tasks to this, I think.